sentry-android-core/src/main/java/io/sentry/android/core/InternalSentrySdk.java

```diff
@@ -148,7 +155,8 @@ public static Map<String, Object> serializeScope(
    *     captured
    */
   @Nullable
-  public static SentryId captureEnvelope(final @NotNull byte[] envelopeData) {
+  public static SentryId captureEnvelope(
+      final @NotNull byte[] envelopeData, final boolean maybeStartNewSession) {
```

Comment:
So if I get it right on the Flutter Android side I just need to check if the error is unhandled and set `maybeStartNewSession` to true in those cases

Reply:
Yes, exactly.

Reply:
Does the envelope contain any extra information we could use instead of `maybeStartNewSession`? 
I see that we have `event.isCrashed()` maybe, having a `event.isUnhandled()` would make sense as well.